tls/datadog_checks/tls/data/conf.yaml.example

```diff
@@ -1,3 +1,5 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
     ## @param allowed_versions - list of strings - optional
```

Comment:
@AlexandreYang  is it expected that `allowed_versions` is both here and in `instances` ?

Reply:
@pducolin Yes, expected https://github.com/DataDog/integrations-core/blob/cd90106f3092aa9af5b0a548e622b47d2a230efb/tls/datadog_checks/tls/tls.py#L98-L101